nossas/bonde
# Feature resync mailchimp

# About
Synchronizing mailchimp in the community context comes from the need to synchronize activists that have not yet been synchronized in the database due to some problem in the initial synchronization moment.

# How to test

1. Open a Mailchimp Tab on Community Settings (`/community/mailchimp`)
2. You should see a new button beside `Salvar` called `Sincronizar`
3. When click these button, should show a notification with request confirmed.

**Depends on server**: https://github.com/nossas/bonde-server/pull/391
